pinc/wordcheck_engine.inc

```diff
@@ -563,44 +565,36 @@ function copy_project_wordcheck_events($from_projectid, $to_projectid)
             (projectid, timestamp, image, round_id, username, suggestions, corrections)
         SELECT '%s', timestamp, image, round_id, username, suggestions, corrections
         FROM wordcheck_events
-        WHERE projectid = '%s'
-        ", mysqli_real_escape_string(DPDatabase::get_connection(), $to_projectid),
-        mysqli_real_escape_string(DPDatabase::get_connection(), $from_projectid));
-
-    mysqli_query(DPDatabase::get_connection(), $sql);
+        WHERE projectid = '%s'",
+        DPDatabase::escape($to_projectid),
+        DPDatabase::escape($from_projectid)
+    );
+    DPDatabase::query($sql);
     // no results for an INSERT clause, so no need to free the results.
 }
 
 // -----------------------------------------------------------------------------
 
 function save_wordcheck_event($projectid, $round, $page, $proofer, $suggestions, $corrections)
 {
-    $suggestion_list = implode("\n", $suggestions);
-
-    $correction_list = array();
-
     // save the corrections in wdiff format
-    foreach($corrections as $correction) {
-        list($orig, $new) = $correction;
+    $correction_list = [];
```

Comment:
is this used anymore?

Reply:
Well spotted. It looks like not. We're inserting and reading it from the DB, but as far as I can see the only user of `load_wordcheck_events` in `show_good_word_suggestions_detail.php` doesn't use it anymore. I'd prefer to do this as a followup MR so as not to change any behavior in this one though.